src/js/migrations.js

```diff
@@ -261,6 +262,40 @@ exports.Migrations= {
     }
   },
 
+  forgetFirstPartySnitches: function (badger) {
+    // this needs to happen before unblockIncorrectlyBlockedDomains
+    console.log("Removing first parties from snitch map...");
+
+    let snitchMap = badger.storage.getBadgerStorageObject("snitch_map"),
+      items = snitchMap.getItemClones();
+
+    for (let domain in items) {
+      let snitches = snitchMap.getItem(domain);
+      let newSnitches = [];
+
+      snitches.forEach(function (snitch) {
+        if (!mdfp.isMultiDomainFirstParty(snitch, domain)) {
```

Comment:
We should use `utils.isThirdPartyDomain()` here instead, to match what we do as part of standard operation.

Reply:
Makes sense! Thanks